examples/relative_move.py

```diff
@@ -1,28 +1,31 @@
+import logging
 from cri_lib import CRIController
 
+logger = logging.getLogger(__name__)  # Retrieves the logger from __init__.py
+
```

Comment:
```suggestion
logger = logging.getLogger(__name__)

```
Here the `__name__` will by `"__main__"`, at least when running this file with `python relative_move.py`.

Reply:
instead of __main__ i have given a name for this relative_move to show in log messages clearly